.github/workflows/build.yml

```diff
@@ -113,84 +104,290 @@ jobs:
-              TF_VAR_password:          "${{ secrets.GOVUKPAAS_PASSWORD  }}"
+  review:
+    name: Review Deployment Process
+    needs: build
+    if: github.ref != 'refs/heads/master' && github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Review Deployment
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: Deploy to PaaS
+          token: ${{ secrets.ACTIONS_API_ACCESS_TOKEN }}
+          inputs: '{"environment": "Review", "sha": "${{ github.sha }}" , "pr": "${{github.event.number}}"  }'
+          ref: ${{ github.event.pull_request.head.ref }}
 
-      - name: Smoke tests
-        if: github.ref == 'refs/heads/master'
-        run: |
-             tests/confidence/healthcheck.sh  "get-into-teaching-app-dev" "${{ steps.sha.outputs.short }}"
-        env:
-             HTTPAUTH_PASSWORD: "${{ secrets.HTTPAUTH_PASSWORD }}"
-             HTTPAUTH_USERNAME: "${{ secrets.HTTPAUTH_USERNAME }}"
+      - name: Wait for Deployment to Review
+        uses: fountainhead/action-wait-for-check@v1.0.0
+        id: wait-for-review
+        with:
+          token: ${{ secrets.ACTIONS_API_ACCESS_TOKEN}}
+          checkName: Deploy Review
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Check if Review Deployment has returned with a failure
+        if: steps.wait-for-review.outputs.conclusion == 'failure'
+        run: exit 1
 
-      - name: Create Sentry release
-        if: success()
-        uses: getsentry/action-release@v1.1.3
+      - name: Review Ready Notification
+        uses: rtCamp/action-slack-notify@master
         env:
-           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-           SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
-           SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
-        with:
-           environment: development
+           SLACK_USERNAME: GiT Review
+           SLACK_CHANNEL: getintoteaching_tech
+           SLACK_COLOR: '#03AC13'
+           SLACK_ICON: https://github.com/rtCamp.png?size=48
+           SLACK_TITLE:  "Review Pending for : ${{github.event.pull_request.title}}"
+           SLACK_MESSAGE: |
+                          Review Application can be found at https://get-into-teaching-${{github.event.number}}.${{env.DOMAIN}}
+                          Pull request can be found at https://github.com/DFE-Digital/${{env.APPLICATION_NAME}}/pull/${{ github.event.number }}
+           SLACK_FOOTER: "${{env.PROJECT}}"
+           MSG_MINIMAL: true
+           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
-      - name: Update Development status
-        if: always()
-        uses: bobheadxi/deployments@v0.4.2
-        with:
-          step: finish
-          token: ${{ secrets.GITHUB_TOKEN}}
-          env: 'Development'
-          status: ${{ job.status }}
-          deployment_id: ${{ steps.deployment.outputs.deployment_id }}
-          ref: ${{ github.sha }}
+      - name: Post PR comment
```

Comment:
Can we try https://github.com/marketplace/actions/sticky-pull-request-comment instead?

Reply:
yup